src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosDbCollectionPhysicalPartitionInfo.cs

```diff
@@ -186,6 +164,23 @@ private static string GenerateAuthToken(string verb, string resourceType, string
             return $"type=master&ver=1.0&sig={signature}";
         }
 
+        private async Task<string> GenerateAccessToken(string host, CancellationToken cancellationToken)
```

Comment:
Can we check that **host** is not null, empty or white space?

Reply:
@abiisnn We are already checking for "host" as not null and empty or white space in line number 101.
Only after the check is successful, GenerateAccessToken function will get invoked.